src/app/Processor/ProcessorDetail/ProcessorDetail.tsx

```diff
@@ -61,7 +61,12 @@ const ProcessorDetail = (props: ProcessorDetailProps): JSX.Element => {
           <Stack hasGutter={true}>
             <StackItem>
               <TextContent>
-                <Text component={TextVariants.h2}>{t("processor.source")}</Text>
+                <Text
+                  component={TextVariants.h2}
+                  ouiaId="processor-source-section"
```

Comment:
To be consistent with the rest of processor details, I think we should use just `source-section`.